test/e2e/app-dir/segment-cache/incremental-opt-in/segment-cache-incremental-opt-in.test.ts

```diff
@@ -10,8 +10,20 @@ describe('segment cache (incremental opt in)', () => {
     return
   }
 
+  function extractPseudoJSONFromFlightResponse(flightText: string) {
+    // This is a cheat that takes advantage of the fact that the roots of the
+    // Flight responses in this test are JSON. This is just a temporary smoke test
+    // until the client part is implemented; we shouldn't rely on this as a
+    // general testing strategy.
+    const match = flightText.match(/^0:(.*)$/m)
+    if (match) {
+      return JSON.parse(match[1])
+    }
+    return null
+  }
+
   // TODO: Replace with e2e test once the client part is implemented
-  it('prefetch responds with 204 if PPR is disabled for a route', async () => {
+  it('route tree prefetch falls through to pre-Segment Cache implementation if PPR is disabled for a route', async () => {
```

Comment:
```suggestion
  it('route tree prefetch falls through to per-segment cache implementation if PPR is disabled for a route', async () => {
```

Reply:
that was actually the intended wording. I meant the "old" (before Segment Cache) implementation. I'll rephrase 😆